secrets.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Secrets of Gorilla Repl
+---
+
+Sourced from Lee Spector (lspector@hampshire.edu), revised October 19, 2016
+
+----
+This document provides tips on using the combination of Leiningen (<http://leiningen.org>) and Gorilla REPL (<http://gorilla-repl.org>) **as a complete Clojure programming environment**. The key features for using Gorilla as a complete environment are:
```

Comment:
Could we maybe expand this a tiny bit to add context:

"This document was written for university students at the Hampshire College taking a course where they likely use Clojure for the first time. It provides [snip] environment. You might find it useful if you're just getting started with Clojure, as it provides an easy workflow for development. The key features [snip]"